strapi/strapi
# Add cloud section on homepage!

### What does it do?

This add a new section on the homepage with info about the new Strapi Cloud as well as a link to go to `cloud.strapi.io`. To make this I also edit the `ContentBox` component from the helper plugin to accept `customIcon` and `decorator` as properties.

### How to test it?

- Run a new Strapi application (make sure of run `yarn setup` before)
- Go to the homepage and see the new cloud section
